schema.graphql

```diff
@@ -46,7 +46,7 @@ type Patron {
   totalHolds: Int!,
   charges: [Charge]!,
   holds: [Hold]!,
-  loans: [Loan]!,
```

Comment:
note to self / TODO I just noticed : looks like there are commas between lines under the Patron type, but not for any other type in the schema ...